### ISSUE TITLE
Add support for posters (cover photo) for signed url videos

### ISSUE THREAD

**Issue by indigonz** [B-Interactive/cloudflare-stream-wordpress#27]
Working on a yoga platform (see attached screenshot) that uses signed url videos when a user is signed in. However, I am unable to set a custom cover photo for each video when working with signed urls and using shortcodes. I would like to set a custom poster, cover photo, for every video because if you can tell from the screenshot all the videos have the same poster photo (light orange background) which makes it look dull

<img width="1145" alt="Screen Shot 2023-06-07 at 14 04 50" src="https://github.com/B-Interactive/cloudflare-stream-wordpress/assets/122932595/8bf95b10-35a5-499e-a90c-afb984733bad">

**Comment by B-Interactive** [comment]
Thank you @indigonz, that's a good example of the issue.  Currently the default poster is being used, which is the first frame of the video.

Initially, which might be more useful to you:
- Being able to specify poster reference time per video (per shortcode).
- Being able to universally specify poster reference time (in admin settings).

**Comment by B-Interactive** [comment]
I ended up implementing both options.  I've made this available in an alpha release here:

https://github.com/B-Interactive/cloudflare-stream-wordpress/releases/tag/1.0.9.4-alpha

**Comment by B-Interactive** [comment]
To set the thumbnail time globally, you'll find the setting in the admin dashboard under Settings > Clouflare Stream.  At the moment, the setting here just takes an integer value, representing seconds.

This can be overridden on a per-video basis using the `postertime` shortcode argument.  The setting here needs you to specify "s" for seconds, and/or "m" for minutes.  For example:
```
[cloudflare_stream uid="some video id" postertime="1m10s"]
```
I might look into making these a bit more consistent when time permits.

Lastly, you can also specify a `posterurl` in the shortcode, which will also override `postertime`.  This allows you to point to any image URL, to use as the thumbnail.  For example:
```
[cloudflare_stream uid="some video id" posterurl="https://example.com/mythumbnail.jpg"]
```

**Comment by B-Interactive** [comment]
I'm going to change the behaviour of the shortcode `postertime`, to be consistent with the global poster time value.  This simplifies validation too.  

Both now only use an integer, that represents a video location in **seconds**.  For example:

Was:
```[cloudflare_stream uid="some video id" postertime="1m10s"]```

Now:
```[cloudflare_stream uid="some video id" postertime="70"]```

**Comment by B-Interactive** [comment]
Completed with 8714df2868d514b71879db796ac57db2980ec86e, and released in [1.0.9.4](https://github.com/B-Interactive/cloudflare-stream-wordpress/releases/tag/1.0.9.4).
